ruby-exercises-Nov-24/Question1/demonstration.rb

```diff
@@ -0,0 +1,20 @@
+require './tester_include.rb'
```

Comment:
No need for 4 separate files to create your module and test it.

```ruby
module HelperMethods

  def titleize string
    dontCapitalize = %w(in the of and or from to)
    string.capitalize.split
      .map { |w| dontCapitalize.include?(w) ? w : w.capitalize }.join ' '
  end

end

class Include
  include HelperMethods

end

class Extend
  extend HelperMethods

end

p Include.new.titleize("hello or something") # using include, you can call the helper method on an instance of the class.
# p Include.titleize("hello or something") -- this will fail because titleize is not a class method
p Extend.titleize("hello or something") # using extends, you can call the helper method on the class itself.
# p Extend.new.titleize("hello or something") -- this will fail because titleize is not a method for instances of the extend class to use
```